spec/app_spec.rb

```diff
@@ -174,4 +174,17 @@ def test_es_check(service_available, status='green', timed_out=false)
       expect(last_response.body).to include File.expand_path(__FILE__)
     end
   end
+
+  describe 'config' do
+    describe 'Elasticsearch client' do
+      subject { Elasticsearch::Model.client }
```

Comment:
Since we didn't really add this, let make sure this isn't a redundant test.  If it really just a new test to existing code, we can keep it.